std/utf.d

```diff
@@ -60,7 +60,7 @@ $(TR $(TD Miscellaneous) $(TD
    +/
 module std.utf;
 
-import std.exception;  // basicExceptionCtors
+import std.exception;  // basicExceptionCtors, assertThrown
```

Comment:
This won't work because tests are extracted. Please import it inside the unittest block.